repository-data/application/src/main/resources/hcm-content/content/documents/administration/value-lists/categories.yaml

```diff
@@ -0,0 +1,27 @@
+/content/documents/administration/value-lists/categories:
```

Comment:
As I understand in the future we may have more site beside LKS. Do you think the categories content should be the same between different sites, if they could be different then we may need to put the ValueList into each site content folder like `/content/documents/lks/valuelists`

Reply:
I have updated the path. I'm not sure how we are going to work with value lists from multiple sites but I assume that we will deal with that later.